<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>861</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>로그인</string>
     </attribute>
     <widget class="QLabel" name="login_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>801</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/의명박6팀(230206~)/등산.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="login_btn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>로그인</string>
      </property>
     </widget>
     <widget class="QLabel" name="login_label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>로그인</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacher_id">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacher_pw">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="pw_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PW</string>
      </property>
     </widget>
     <widget class="QLabel" name="teacher_label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;선생님용&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sign_up">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QLabel" name="login_label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="combo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>교사</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>회원가입</string>
     </attribute>
     <widget class="QLabel" name="login_label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>801</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/의명박6팀(230206~)/등산.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="check_ID_btn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>중복확인</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PW</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ID">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ps_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="check_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="register_signup_btn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>이름</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>교사</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="join_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>문제</string>
     </attribute>
     <widget class="QLabel" name="Map_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>551</width>
        <height>791</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/의명박6팀(230206~)/100대명산지도.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="background_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-170</y>
        <width>1201</width>
        <height>1071</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/의명박6팀(230206~)/Background.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>470</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/경상북도.png</normaloff>../../Desktop/의명박6팀(230206~)/경상북도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="shortcut">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="jeonbuk_btn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>470</y>
        <width>157</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/전라북도.png</normaloff>../../Desktop/의명박6팀(230206~)/전라북도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="jeonnam_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>580</y>
        <width>157</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/전라남도.png</normaloff>../../Desktop/의명박6팀(230206~)/전라남도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="gyeongnam_btn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>550</y>
        <width>157</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/경상남도.png</normaloff>../../Desktop/의명박6팀(230206~)/경상남도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="gyeongbuk_btn2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>470</y>
        <width>151</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/경상북도.png</normaloff>../../Desktop/의명박6팀(230206~)/경상북도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="gyeongbuk_btn">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>370</y>
        <width>157</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/경상북도.png</normaloff>../../Desktop/의명박6팀(230206~)/경상북도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="chungnam_btn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>161</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/충청남도.png</normaloff>../../Desktop/의명박6팀(230206~)/충청남도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="chungbuk_btn">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>157</width>
        <height>48</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/충청북도.png</normaloff>../../Desktop/의명박6팀(230206~)/충청북도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="gyeonggi_btn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>167</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/서울경기.png</normaloff>../../Desktop/의명박6팀(230206~)/서울경기.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="gangwon_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>149</y>
        <width>127</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/강원도.png</normaloff>../../Desktop/의명박6팀(230206~)/강원도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="jeju_btn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>750</y>
        <width>128</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/의명박6팀(230206~)/제주도.png</normaloff>../../Desktop/의명박6팀(230206~)/제주도.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>800</width>
        <height>900</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="O_btn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>610</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>O</string>
      </property>
     </widget>
     <widget class="QPushButton" name="X_btn">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>610</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_btn">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>700</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>문제 저장</string>
      </property>
     </widget>
     <widget class="QLabel" name="quiz_label">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>70</y>
        <width>300</width>
        <height>491</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <zorder>background_label</zorder>
     <zorder>Map_label</zorder>
     <zorder>pushButton</zorder>
     <zorder>jeonbuk_btn</zorder>
     <zorder>jeonnam_btn</zorder>
     <zorder>gyeongnam_btn</zorder>
     <zorder>gyeongbuk_btn2</zorder>
     <zorder>gyeongbuk_btn</zorder>
     <zorder>chungnam_btn</zorder>
     <zorder>chungbuk_btn</zorder>
     <zorder>gyeonggi_btn</zorder>
     <zorder>gangwon_btn</zorder>
     <zorder>jeju_btn</zorder>
     <zorder>O_btn</zorder>
     <zorder>X_btn</zorder>
     <zorder>save_btn</zorder>
     <zorder>quiz_label</zorder>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>점수현황</string>
     </attribute>
     <widget class="QLabel" name="login_label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>801</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/의명박6팀(230206~)/등산.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Q&amp;&amp;A</string>
     </attribute>
     <widget class="QLabel" name="login_label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>801</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/의명박6팀(230206~)/등산.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Q &amp; A</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="qna_tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>711</width>
        <height>431</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>순 번</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>날 짜</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>이 름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>질 문</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>답 변</string>
       </property>
      </column>
      <item row="0" column="3">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="answer_lineedit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>580</y>
        <width>711</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="send_btn">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>580</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>입 력</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>실시간 상담</string>
     </attribute>
     <widget class="QLabel" name="login_label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>801</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/의명박6팀(230206~)/등산.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>451</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>650</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>650</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>전 송</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>60</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>상담 메시지</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
